api-admin/src/main/java/bio/terra/pearl/api/admin/service/family/FamilyExtService.java

```diff
@@ -113,4 +117,37 @@ public List<DataChangeRecord> listChangeRecords(
 
     return familyService.findDataChangeRecordsByFamilyId(family.getId());
   }
+
+  @EnforcePortalStudyEnvPermission(permission = "participant_data_edit")
+  public Family create(PortalStudyEnvAuthContext authContext, Family family, String justification) {
+    if (!family.getStudyEnvironmentId().equals(authContext.getStudyEnvironment().getId())) {
+      throw new NotFoundException("Study environment not found");
+    }
+
+    Enrollee proband =
+        enrolleeService
+            .find(family.getProbandEnrolleeId())
+            .orElseThrow(() -> new NotFoundException("Proband not found"));
+
+    if (!proband.getStudyEnvironmentId().equals(family.getStudyEnvironmentId())) {
+      throw new NotFoundException("Proband not found");
+    }
+
+    AdminUser user = authContext.getOperator();
```

Comment:
I don't think you need this `user` variable, and if you do, please call it `operator`